src/js/game/GameController.js

```diff
@@ -1198,6 +1203,22 @@ class GameController {
     });
   }
 
+  checkConflict(position, placementPlane) {
+    var conflict = false;
+    this.levelEntity.entityMap.forEach( workingEntity => {
+      if (Position.equals(workingEntity.position, position)) {
+        conflict = true;
+      }
+    });
```

Comment:
Nit:

```js
const entityConflict = this.levelEntity.entityMap.some(entity => {
  return Position.equals(entity.position, position);
});
```

Reply:
Uncaught TypeError: this.levelEntity.entityMap.some is not a function

😭